ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -47,6 +47,9 @@ else
             "${SHARED_DIR}/test-list" \
             "root@${IP}:/tmp/test-list"
         echo "### Running tests"
+        timeout \
+        --kill-after 10m \
+        180m \
```

Comment:
Considering that usually e2e-metal-* builds execution times are in average 90m (~[1h, 2h]), I'd keep a timeout of 120m max to be on the safe side, considering that the global CI timeout is 4h

Reply:
After adding new green tests for serial and parallel suite, if you think that 2 hours will be sufficient, of course, I can change it to 120m

Reply:
Well, the problem now is about the possibility to gather some (even partial) results when some (bad) long-standing tests will delay the execution time over the 4h. If the `setup time + current test timeout` will be greater than 4h then the test step will be again interrupted and no artifacts will be collected. With a `setup time + current test timeout` lower than 4h then if a test will get stuck we'll be able to detect it.